packages/mobile/src/screens/app-screen/ExploreTabScreen.tsx

```diff
@@ -68,29 +73,43 @@ const smartCollections = [
 ]
 
 export const ExploreTabScreen =
-  createAppTabScreenStack<ExploreTabScreenParamList>((Stack) => (
-    <>
-      <Stack.Screen name='Explore' component={ExploreScreen} />
-      <Stack.Screen name='PremiumTracks' component={PremiumTracksScreen} />
-      <Stack.Screen name='LetThemDJ' component={LetThemDJScreen} />
-      <Stack.Screen name='TopAlbums' component={TopAlbumsScreen} />
-      <Stack.Screen
-        name='TrendingPlaylists'
-        component={TrendingPlaylistsScreen}
-      />
-      <Stack.Screen
-        name='TrendingUnderground'
-        component={TrendingUndergroundScreen}
-      />
-      {smartCollections.map((collection) => (
-        <Stack.Screen name={collection.screen} key={collection.screen}>
-          {() => <SmartCollectionScreen smartCollection={collection} />}
-        </Stack.Screen>
-      ))}
-      {moodCollections.map((collection) => (
-        <Stack.Screen name={collection.screen} key={collection.screen}>
-          {() => <MoodCollectionScreen collection={collection} />}
-        </Stack.Screen>
-      ))}
-    </>
-  ))
+  createAppTabScreenStack<ExploreTabScreenParamList>((Stack) => {
+    const { isEnabled: isSearchExploreEnabled } = useFeatureFlag(
```

Comment:
so technically i think this is not a component, since it doesn't have traditional props. can we confirm if we get a hook warning in the console when running? it's an easy change if so, which is to create a component outside of this that recieves Stack as a prop, and then just do

```
createAppTabScreenStack((Stack) => <TheNewComponent Stack={Stack} />
```